commons/src/main/java/org/infinispan/commons/marshall/WrappedByteArray.java

```diff
@@ -16,6 +17,7 @@
  * @author wburns
  * @since 9.0
  */
+@ProtoDoc(ProtoStreamTypeIds.WRAPPED_BYTE_ARRAY)
```

Comment:
This makes me want to add a proper ```@ProtoTypeId``` annotation, but is good ATM :)

Reply:
`@ProtoTypeId` would be very nice for the future

Reply:
The future is near https://issues.jboss.org/browse/IPROTO-111
There's also a PR for upgrading protostream.